dist/profile/manifests/openvpn.pp

```diff
@@ -82,6 +85,9 @@
     ensure  => 'file',
     owner   => 'root',
     group   => 'root',
+    require => [
+      File['/etc/netplan/'],
```

Comment:
@dduportal we need to verify it there even with the requirement on the same folder above?

Reply:
The doc for puppet 6 specifies that the requirement to parent folder is not mandatory: https://puppet.com/docs/puppet/6/types/file.html#file-description (read the `autorequire` part).

I'm still not sure why, but in some cases it fails so better to make it explicit.

(using the `require` attribute describe the dependency, and Puppet does the test for us)